src/zenml/integrations/integration.py

```diff
@@ -61,6 +61,7 @@ def check_installation(cls) -> bool:
         Returns:
             True if all required packages are installed, False otherwise.
         """
+        cls.check_system()
```

Comment:
I'm not sure if it's enough to call this here. I think `zenml integration install tensorflow -f` for instance would never reach this because `src.zenml.cli.integration.install` does this:
```
if force or not integration_registry.is_installed(
    integration_name
):
    ... # install integration
```

Reply:
perhaps we could call this in `IntegrationMeta.__new__` instead so the requirements are overridden right when the class is created?